autogen.sh

```diff
@@ -3,6 +3,15 @@
 rm -rf autom4te.cache
 mkdir -p config/m4 config/aux external/jemalloc
 
+GIT_FOUND=1
+command git --help 2>&1 > /dev/null || { echo "Git not found, not retrieving external packages"; export GIT_FOUND=0; }
```

Comment:
1. Please avoid long lines
2. You don't really have to export since this is an internal variable
3. Please add more descriptive error saying that you not going to install jemalloc